content/en/news/releases/1.13.x/announcing-1.13.3/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Announcing Istio 1.13.3
+linktitle: 1.13.3
+subtitle: Patch Release
+description: Istio 1.13.3 patch release.
+publishdate: 2022-04-18
+release: 1.13.3
+aliases:
+    - /news/announcing-1.13.3
+---
+
+This release contains bug fixes to improve robustness and some additional configuration support.
+This release note describes what's different between Istio 1.13.2 and 1.13.3.
+
+{{< relnote >}}
+
+## Changes
+
+- **Added** support for skipping the initial installation of CNI entirely.
+
+- **Added** values to the Istio Pilot Helm charts for configuring affinity rules and toleration on the Deployment.
+  Can be used for better placement of Istio pilot workloads.
```

Comment:
```suggestion
- **Added** values to the istiod Helm charts for configuring affinity rules and toleration on the Deployment. This can be used for better placement of istiod instances.
```